[2025-02-16 16:02:48,102:INFO:main:Welcome to our custom logging data science]
[2025-02-16 16:03:30,464:INFO:main:Welcome to our custom logging data science]
[2025-02-16 16:03:30,465:INFO:main:>>>>>stageData Ingestion Stage started <<<<<]
[2025-02-16 16:03:30,466:INFO:common:yaml file: config/config.yaml loaded successfully]
[2025-02-16 16:03:30,466:INFO:common:yaml file: params.yaml loaded successfully]
[2025-02-16 16:03:30,466:INFO:common:yaml file: schema.yaml loaded successfully]
[2025-02-16 16:03:30,467:INFO:common:created directory at: artifacts]
[2025-02-16 16:03:30,467:INFO:common:created directory at: artifacts/data_ingestion]
[2025-02-16 16:03:30,995:INFO:data_ingestion:artifacts/data_ingestion/data.zip download! with following info: 
Connection: close
Content-Length: 23329
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "c69888a4ae59bc5a893392785a938ccd4937981c06ba8a9d6a21aa52b4ab5b6e"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 195D:1EB9D8:1157B33:137BE50:67B24C58
Accept-Ranges: bytes
Date: Sun, 16 Feb 2025 21:03:30 GMT
Via: 1.1 varnish
X-Served-By: cache-chi-kigq8000108-CHI
X-Cache: HIT
X-Cache-Hits: 0
X-Timer: S1739739811.923288,VS0,VE67
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 6faf95bfde7c8ff3033b61b6381a87806b202988
Expires: Sun, 16 Feb 2025 21:08:30 GMT
Source-Age: 0

]
[2025-02-16 16:03:30,997:INFO:main:>>>>>stageData Ingestion Stage completed <<<<<]
[2025-02-19 21:03:55,701:INFO:common:yaml file: config/config.yaml loaded successfully]
[2025-02-19 21:03:55,707:INFO:common:yaml file: params.yaml loaded successfully]
[2025-02-19 21:03:55,710:INFO:common:yaml file: schema.yaml loaded successfully]
[2025-02-19 21:03:55,714:INFO:common:created directory at: artifacts]
[2025-02-23 15:00:23,014:INFO:common:yaml file: config/config.yaml loaded successfully]
[2025-02-23 15:00:23,048:INFO:common:yaml file: params.yaml loaded successfully]
[2025-02-23 15:00:23,055:INFO:common:yaml file: schema.yaml loaded successfully]
[2025-02-23 15:00:23,059:INFO:common:created directory at: artifacts]
[2025-02-23 15:00:23,059:INFO:common:created directory at: artifacts/data_ingestion]
[2025-02-23 15:00:23,060:INFO:2639782589:File already exists]
[2025-02-23 15:07:50,667:INFO:common:yaml file: config/config.yaml loaded successfully]
[2025-02-23 15:07:50,669:INFO:common:yaml file: params.yaml loaded successfully]
[2025-02-23 15:07:50,671:INFO:common:yaml file: schema.yaml loaded successfully]
[2025-02-23 15:07:50,673:INFO:common:created directory at: artifacts]
[2025-02-23 15:10:37,924:INFO:common:yaml file: config/config.yaml loaded successfully]
[2025-02-23 15:10:37,926:INFO:common:yaml file: params.yaml loaded successfully]
[2025-02-23 15:10:37,929:INFO:common:yaml file: schema.yaml loaded successfully]
[2025-02-23 15:10:37,930:INFO:common:created directory at: artifacts]
[2025-02-23 15:14:14,175:INFO:common:yaml file: config/config.yaml loaded successfully]
[2025-02-23 15:14:14,177:INFO:common:yaml file: params.yaml loaded successfully]
[2025-02-23 15:14:14,181:INFO:common:yaml file: schema.yaml loaded successfully]
[2025-02-23 15:14:14,182:INFO:common:created directory at: artifacts]
[2025-02-23 15:14:14,183:INFO:common:created directory at: artifacts/data_validation]
[2025-02-23 15:14:24,704:INFO:common:yaml file: config/config.yaml loaded successfully]
[2025-02-23 15:14:24,706:INFO:common:yaml file: params.yaml loaded successfully]
[2025-02-23 15:14:24,709:INFO:common:yaml file: schema.yaml loaded successfully]
[2025-02-23 15:14:24,710:INFO:common:created directory at: artifacts]
[2025-02-23 15:14:24,711:INFO:common:created directory at: artifacts/data_validation]
[2025-02-23 15:15:41,510:INFO:common:yaml file: config/config.yaml loaded successfully]
[2025-02-23 15:15:41,512:INFO:common:yaml file: params.yaml loaded successfully]
[2025-02-23 15:15:41,515:INFO:common:yaml file: schema.yaml loaded successfully]
[2025-02-23 15:15:41,516:INFO:common:created directory at: artifacts]
[2025-02-23 15:15:41,517:INFO:common:created directory at: artifacts/data_validation]
[2025-02-23 15:30:23,872:INFO:main:Welcome to our custom logging data science]
[2025-02-23 15:30:23,872:INFO:main:>>>>>stageData Ingestion Stage started <<<<<]
[2025-02-23 15:30:23,873:INFO:common:yaml file: config/config.yaml loaded successfully]
[2025-02-23 15:30:23,874:INFO:common:yaml file: params.yaml loaded successfully]
[2025-02-23 15:30:23,875:INFO:common:yaml file: schema.yaml loaded successfully]
[2025-02-23 15:30:23,876:INFO:common:created directory at: artifacts]
[2025-02-23 15:30:23,876:INFO:common:created directory at: artifacts/data_ingestion]
[2025-02-23 15:30:23,876:INFO:data_ingestion:File already exists]
[2025-02-23 15:30:23,878:INFO:main:>>>>>stageData Ingestion Stage completed <<<<<]
[2025-02-23 15:30:23,878:INFO:main:>>>>>> stage Data Validation stage started <<<<<<]
[2025-02-23 15:30:23,878:ERROR:main:name 'DataValidationTrainingPipeline' is not defined]
Traceback (most recent call last):
  File "/Users/ruilu/datascienceproject/main.py", line 21, in <module>
    data_ingestion = DataValidationTrainingPipeline()
NameError: name 'DataValidationTrainingPipeline' is not defined
[2025-02-23 15:31:05,883:INFO:main:Welcome to our custom logging data science]
[2025-02-23 15:31:05,883:INFO:main:>>>>>stageData Ingestion Stage started <<<<<]
[2025-02-23 15:31:05,884:INFO:common:yaml file: config/config.yaml loaded successfully]
[2025-02-23 15:31:05,884:INFO:common:yaml file: params.yaml loaded successfully]
[2025-02-23 15:31:05,886:INFO:common:yaml file: schema.yaml loaded successfully]
[2025-02-23 15:31:05,886:INFO:common:created directory at: artifacts]
[2025-02-23 15:31:05,886:INFO:common:created directory at: artifacts/data_ingestion]
[2025-02-23 15:31:05,886:INFO:data_ingestion:File already exists]
[2025-02-23 15:31:05,888:INFO:main:>>>>>stageData Ingestion Stage completed <<<<<]
[2025-02-23 15:31:05,888:INFO:main:>>>>>> stage Data Validation stage started <<<<<<]
[2025-02-23 15:31:05,888:ERROR:main:name 'DataValidationTrainingPipeline' is not defined]
Traceback (most recent call last):
  File "/Users/ruilu/datascienceproject/main.py", line 21, in <module>
    data_ingestion = DataValidationTrainingPipeline()
NameError: name 'DataValidationTrainingPipeline' is not defined
[2025-02-23 15:31:40,444:INFO:main:Welcome to our custom logging data science]
[2025-02-23 15:31:40,444:INFO:main:>>>>>stageData Ingestion Stage started <<<<<]
[2025-02-23 15:31:40,446:INFO:common:yaml file: config/config.yaml loaded successfully]
[2025-02-23 15:31:40,446:INFO:common:yaml file: params.yaml loaded successfully]
[2025-02-23 15:31:40,447:INFO:common:yaml file: schema.yaml loaded successfully]
[2025-02-23 15:31:40,448:INFO:common:created directory at: artifacts]
[2025-02-23 15:31:40,448:INFO:common:created directory at: artifacts/data_ingestion]
[2025-02-23 15:31:40,448:INFO:data_ingestion:File already exists]
[2025-02-23 15:31:40,449:INFO:main:>>>>>stageData Ingestion Stage completed <<<<<]
[2025-02-23 15:31:40,449:INFO:main:>>>>>> stage Data Validation stage started <<<<<<]
[2025-02-23 15:31:40,451:INFO:common:yaml file: config/config.yaml loaded successfully]
[2025-02-23 15:31:40,451:INFO:common:yaml file: params.yaml loaded successfully]
[2025-02-23 15:31:40,453:INFO:common:yaml file: schema.yaml loaded successfully]
[2025-02-23 15:31:40,453:INFO:common:created directory at: artifacts]
[2025-02-23 15:31:40,453:INFO:common:created directory at: artifacts/data_validation]
[2025-02-23 15:31:40,458:INFO:main:>>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2025-02-23 15:47:04,627:INFO:common:yaml file: config/config.yaml loaded successfully]
[2025-02-23 15:47:04,629:INFO:common:yaml file: params.yaml loaded successfully]
[2025-02-23 15:47:04,632:INFO:common:yaml file: schema.yaml loaded successfully]
[2025-02-23 15:47:04,633:INFO:common:created directory at: artifacts]
[2025-02-23 15:47:04,634:INFO:common:created directory at: artifacts/data_transformation]
[2025-02-23 15:47:04,662:INFO:741865018:Splited data into training and test sets]
[2025-02-23 15:47:04,662:INFO:741865018:(1199, 12)]
[2025-02-23 15:47:04,663:INFO:741865018:(400, 12)]
[2025-02-23 15:59:57,822:INFO:main:Welcome to our custom logging data science]
[2025-02-23 15:59:57,823:INFO:main:>>>>>stageData Ingestion Stage started <<<<<]
[2025-02-23 15:59:57,824:INFO:common:yaml file: config/config.yaml loaded successfully]
[2025-02-23 15:59:57,825:INFO:common:yaml file: params.yaml loaded successfully]
[2025-02-23 15:59:57,826:INFO:common:yaml file: schema.yaml loaded successfully]
[2025-02-23 15:59:57,826:INFO:common:created directory at: artifacts]
[2025-02-23 15:59:57,826:INFO:common:created directory at: artifacts/data_ingestion]
[2025-02-23 15:59:57,826:INFO:data_ingestion:File already exists]
[2025-02-23 15:59:57,829:INFO:main:>>>>>stageData Ingestion Stage completed <<<<<]
[2025-02-23 15:59:57,829:INFO:main:>>>>>> stage Data Validation stage started <<<<<<]
[2025-02-23 15:59:57,830:INFO:common:yaml file: config/config.yaml loaded successfully]
[2025-02-23 15:59:57,830:INFO:common:yaml file: params.yaml loaded successfully]
[2025-02-23 15:59:57,832:INFO:common:yaml file: schema.yaml loaded successfully]
[2025-02-23 15:59:57,832:INFO:common:created directory at: artifacts]
[2025-02-23 15:59:57,832:INFO:common:created directory at: artifacts/data_validation]
[2025-02-23 15:59:57,873:INFO:main:>>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2025-02-23 15:59:57,873:INFO:main:>>>>>> stage Data Transformation stage started <<<<<<]
[2025-02-23 15:59:57,874:INFO:common:yaml file: config/config.yaml loaded successfully]
[2025-02-23 15:59:57,875:INFO:common:yaml file: params.yaml loaded successfully]
[2025-02-23 15:59:57,880:INFO:common:yaml file: schema.yaml loaded successfully]
[2025-02-23 15:59:57,880:INFO:common:created directory at: artifacts]
[2025-02-23 15:59:57,880:INFO:common:created directory at: artifacts/data_transformation]
[2025-02-23 15:59:57,880:INFO:main:>>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2025-02-23 17:43:22,433:INFO:common:yaml file: config/config.yaml loaded successfully]
[2025-02-23 17:43:22,435:INFO:common:yaml file: params.yaml loaded successfully]
[2025-02-23 17:43:22,439:INFO:common:yaml file: schema.yaml loaded successfully]
[2025-02-23 17:43:22,440:INFO:common:created directory at: artifacts]
[2025-02-23 18:13:46,330:INFO:common:yaml file: config/config.yaml loaded successfully]
[2025-02-23 18:13:46,336:INFO:common:yaml file: params.yaml loaded successfully]
[2025-02-23 18:13:46,339:INFO:common:yaml file: schema.yaml loaded successfully]
[2025-02-23 18:13:46,342:INFO:common:created directory at: artifacts]
[2025-02-23 18:13:59,332:INFO:common:yaml file: config/config.yaml loaded successfully]
[2025-02-23 18:13:59,335:INFO:common:yaml file: params.yaml loaded successfully]
[2025-02-23 18:13:59,338:INFO:common:yaml file: schema.yaml loaded successfully]
[2025-02-23 18:13:59,339:INFO:common:created directory at: artifacts]
[2025-02-23 18:16:45,073:INFO:common:yaml file: config/config.yaml loaded successfully]
[2025-02-23 18:16:45,075:INFO:common:yaml file: params.yaml loaded successfully]
[2025-02-23 18:16:45,078:INFO:common:yaml file: schema.yaml loaded successfully]
[2025-02-23 18:16:45,079:INFO:common:created directory at: artifacts]
[2025-02-23 18:16:45,081:INFO:common:created directory at: artifacts/model_trainer]
[2025-02-26 21:50:27,010:INFO:main:Welcome to our custom logging data science]
[2025-02-26 21:50:27,010:INFO:main:>>>>>stageData Ingestion Stage started <<<<<]
[2025-02-26 21:50:27,012:INFO:common:yaml file: config/config.yaml loaded successfully]
[2025-02-26 21:50:27,014:INFO:common:yaml file: params.yaml loaded successfully]
[2025-02-26 21:50:27,016:INFO:common:yaml file: schema.yaml loaded successfully]
[2025-02-26 21:50:27,017:INFO:common:created directory at: artifacts]
[2025-02-26 21:50:27,017:INFO:common:created directory at: artifacts/data_ingestion]
[2025-02-26 21:50:27,017:INFO:data_ingestion:File already exists]
[2025-02-26 21:50:27,019:INFO:main:>>>>>stageData Ingestion Stage completed <<<<<]
[2025-02-26 21:50:27,020:INFO:main:>>>>>> stage Data Validation stage started <<<<<<]
[2025-02-26 21:50:27,021:INFO:common:yaml file: config/config.yaml loaded successfully]
[2025-02-26 21:50:27,022:INFO:common:yaml file: params.yaml loaded successfully]
[2025-02-26 21:50:27,023:INFO:common:yaml file: schema.yaml loaded successfully]
[2025-02-26 21:50:27,023:INFO:common:created directory at: artifacts]
[2025-02-26 21:50:27,023:INFO:common:created directory at: artifacts/data_validation]
[2025-02-26 21:50:27,032:INFO:main:>>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2025-02-26 21:50:27,032:INFO:main:>>>>>> stage Data Transformation stage started <<<<<<]
[2025-02-26 21:50:27,034:INFO:common:yaml file: config/config.yaml loaded successfully]
[2025-02-26 21:50:27,035:INFO:common:yaml file: params.yaml loaded successfully]
[2025-02-26 21:50:27,036:INFO:common:yaml file: schema.yaml loaded successfully]
[2025-02-26 21:50:27,036:INFO:common:created directory at: artifacts]
[2025-02-26 21:50:27,036:INFO:common:created directory at: artifacts/data_transformation]
[2025-02-26 21:50:27,036:INFO:main:>>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2025-02-26 21:50:27,036:INFO:main:>>>>>> stage Model Trainer stage started <<<<<<]
[2025-02-26 21:50:27,038:INFO:common:yaml file: config/config.yaml loaded successfully]
[2025-02-26 21:50:27,038:INFO:common:yaml file: params.yaml loaded successfully]
[2025-02-26 21:50:27,039:INFO:common:yaml file: schema.yaml loaded successfully]
[2025-02-26 21:50:27,039:INFO:common:created directory at: artifacts]
[2025-02-26 21:50:27,039:ERROR:main:'ConfigurationManager' object has no attribute 'get_model_trainer_config']
Traceback (most recent call last):
  File "/Users/ruilu/datascienceproject/main.py", line 46, in <module>
    data_ingestion.initiate_model_training()
  File "/Users/ruilu/datascienceproject/src/datascience/pipeline/model_trainer_pipeline.py", line 14, in initiate_model_training
    model_trainer_config = config.get_model_trainer_config()
AttributeError: 'ConfigurationManager' object has no attribute 'get_model_trainer_config'
[2025-02-26 21:51:29,002:INFO:main:Welcome to our custom logging data science]
[2025-02-26 21:51:29,002:INFO:main:>>>>>stageData Ingestion Stage started <<<<<]
[2025-02-26 21:51:29,004:INFO:common:yaml file: config/config.yaml loaded successfully]
[2025-02-26 21:51:29,005:INFO:common:yaml file: params.yaml loaded successfully]
[2025-02-26 21:51:29,006:INFO:common:yaml file: schema.yaml loaded successfully]
[2025-02-26 21:51:29,006:INFO:common:created directory at: artifacts]
[2025-02-26 21:51:29,006:INFO:common:created directory at: artifacts/data_ingestion]
[2025-02-26 21:51:29,007:INFO:data_ingestion:File already exists]
[2025-02-26 21:51:29,009:INFO:main:>>>>>stageData Ingestion Stage completed <<<<<]
[2025-02-26 21:51:29,009:INFO:main:>>>>>> stage Data Validation stage started <<<<<<]
[2025-02-26 21:51:29,010:INFO:common:yaml file: config/config.yaml loaded successfully]
[2025-02-26 21:51:29,011:INFO:common:yaml file: params.yaml loaded successfully]
[2025-02-26 21:51:29,012:INFO:common:yaml file: schema.yaml loaded successfully]
[2025-02-26 21:51:29,012:INFO:common:created directory at: artifacts]
[2025-02-26 21:51:29,013:INFO:common:created directory at: artifacts/data_validation]
[2025-02-26 21:51:29,017:INFO:main:>>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2025-02-26 21:51:29,017:INFO:main:>>>>>> stage Data Transformation stage started <<<<<<]
[2025-02-26 21:51:29,019:INFO:common:yaml file: config/config.yaml loaded successfully]
[2025-02-26 21:51:29,019:INFO:common:yaml file: params.yaml loaded successfully]
[2025-02-26 21:51:29,021:INFO:common:yaml file: schema.yaml loaded successfully]
[2025-02-26 21:51:29,021:INFO:common:created directory at: artifacts]
[2025-02-26 21:51:29,021:INFO:common:created directory at: artifacts/data_transformation]
[2025-02-26 21:51:29,021:INFO:main:>>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2025-02-26 21:51:29,021:INFO:main:>>>>>> stage Model Trainer stage started <<<<<<]
[2025-02-26 21:51:29,023:INFO:common:yaml file: config/config.yaml loaded successfully]
[2025-02-26 21:51:29,023:INFO:common:yaml file: params.yaml loaded successfully]
[2025-02-26 21:51:29,025:INFO:common:yaml file: schema.yaml loaded successfully]
[2025-02-26 21:51:29,025:INFO:common:created directory at: artifacts]
[2025-02-26 21:51:29,025:INFO:common:created directory at: artifacts/model_trainer]
[2025-02-26 21:51:29,044:INFO:main:>>>>>> stage Model Trainer stage completed <<<<<<

x==========x]
[2025-03-02 11:01:41,820:INFO:common:yaml file: config/config.yaml loaded successfully]
[2025-03-02 11:01:41,829:INFO:common:yaml file: params.yaml loaded successfully]
[2025-03-02 11:01:41,835:INFO:common:yaml file: schema.yaml loaded successfully]
[2025-03-02 11:01:41,838:INFO:common:created directory at: artifacts]
[2025-03-02 11:01:41,841:INFO:common:created directory at: artifacts/model_evaluation]
[2025-03-02 11:01:43,427:INFO:common:json file saved at: artifacts/model_evaluation/metrics.json]
[2025-03-02 11:29:28,376:INFO:main:Welcome to our custom logging data science]
[2025-03-02 11:29:28,377:INFO:main:>>>>>stageData Ingestion Stage started <<<<<]
[2025-03-02 11:29:28,379:INFO:common:yaml file: config/config.yaml loaded successfully]
[2025-03-02 11:29:28,380:INFO:common:yaml file: params.yaml loaded successfully]
[2025-03-02 11:29:28,381:INFO:common:yaml file: schema.yaml loaded successfully]
[2025-03-02 11:29:28,382:INFO:common:created directory at: artifacts]
[2025-03-02 11:29:28,382:INFO:common:created directory at: artifacts/data_ingestion]
[2025-03-02 11:29:28,382:INFO:data_ingestion:File already exists]
[2025-03-02 11:29:28,384:INFO:main:>>>>>stageData Ingestion Stage completed <<<<<]
[2025-03-02 11:29:28,384:INFO:main:>>>>>> stage Data Validation stage started <<<<<<]
[2025-03-02 11:29:28,386:INFO:common:yaml file: config/config.yaml loaded successfully]
[2025-03-02 11:29:28,387:INFO:common:yaml file: params.yaml loaded successfully]
[2025-03-02 11:29:28,388:INFO:common:yaml file: schema.yaml loaded successfully]
[2025-03-02 11:29:28,388:INFO:common:created directory at: artifacts]
[2025-03-02 11:29:28,388:INFO:common:created directory at: artifacts/data_validation]
[2025-03-02 11:29:28,393:INFO:main:>>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2025-03-02 11:29:28,393:INFO:main:>>>>>> stage Data Transformation stage started <<<<<<]
[2025-03-02 11:29:28,396:INFO:common:yaml file: config/config.yaml loaded successfully]
[2025-03-02 11:29:28,396:INFO:common:yaml file: params.yaml loaded successfully]
[2025-03-02 11:29:28,398:INFO:common:yaml file: schema.yaml loaded successfully]
[2025-03-02 11:29:28,398:INFO:common:created directory at: artifacts]
[2025-03-02 11:29:28,398:INFO:common:created directory at: artifacts/data_transformation]
[2025-03-02 11:29:28,398:INFO:main:>>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2025-03-02 11:29:28,398:INFO:main:>>>>>> stage Model Trainer stage started <<<<<<]
[2025-03-02 11:29:28,400:INFO:common:yaml file: config/config.yaml loaded successfully]
[2025-03-02 11:29:28,400:INFO:common:yaml file: params.yaml loaded successfully]
[2025-03-02 11:29:28,401:INFO:common:yaml file: schema.yaml loaded successfully]
[2025-03-02 11:29:28,401:INFO:common:created directory at: artifacts]
[2025-03-02 11:29:28,402:INFO:common:created directory at: artifacts/model_trainer]
[2025-03-02 11:29:28,414:INFO:main:>>>>>> stage Model Trainer stage completed <<<<<<

x==========x]
[2025-03-02 11:29:28,414:INFO:main:>>>>>> stage Model evaluation stage started <<<<<<]
[2025-03-02 11:29:28,414:ERROR:main:'ModelEvaluationPipeline' object has no attribute 'initiate_model_evaluation']
Traceback (most recent call last):
  File "/Users/ruilu/datascienceproject/main.py", line 57, in <module>
    data_ingestion.initiate_model_evaluation()
AttributeError: 'ModelEvaluationPipeline' object has no attribute 'initiate_model_evaluation'
[2025-03-02 11:29:47,686:INFO:main:Welcome to our custom logging data science]
[2025-03-02 11:29:47,687:INFO:main:>>>>>stageData Ingestion Stage started <<<<<]
[2025-03-02 11:29:47,689:INFO:common:yaml file: config/config.yaml loaded successfully]
[2025-03-02 11:29:47,690:INFO:common:yaml file: params.yaml loaded successfully]
[2025-03-02 11:29:47,691:INFO:common:yaml file: schema.yaml loaded successfully]
[2025-03-02 11:29:47,691:INFO:common:created directory at: artifacts]
[2025-03-02 11:29:47,691:INFO:common:created directory at: artifacts/data_ingestion]
[2025-03-02 11:29:47,691:INFO:data_ingestion:File already exists]
[2025-03-02 11:29:47,693:INFO:main:>>>>>stageData Ingestion Stage completed <<<<<]
[2025-03-02 11:29:47,693:INFO:main:>>>>>> stage Data Validation stage started <<<<<<]
[2025-03-02 11:29:47,695:INFO:common:yaml file: config/config.yaml loaded successfully]
[2025-03-02 11:29:47,696:INFO:common:yaml file: params.yaml loaded successfully]
[2025-03-02 11:29:47,697:INFO:common:yaml file: schema.yaml loaded successfully]
[2025-03-02 11:29:47,698:INFO:common:created directory at: artifacts]
[2025-03-02 11:29:47,698:INFO:common:created directory at: artifacts/data_validation]
[2025-03-02 11:29:47,702:INFO:main:>>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2025-03-02 11:29:47,702:INFO:main:>>>>>> stage Data Transformation stage started <<<<<<]
[2025-03-02 11:29:47,704:INFO:common:yaml file: config/config.yaml loaded successfully]
[2025-03-02 11:29:47,705:INFO:common:yaml file: params.yaml loaded successfully]
[2025-03-02 11:29:47,706:INFO:common:yaml file: schema.yaml loaded successfully]
[2025-03-02 11:29:47,706:INFO:common:created directory at: artifacts]
[2025-03-02 11:29:47,706:INFO:common:created directory at: artifacts/data_transformation]
[2025-03-02 11:29:47,706:INFO:main:>>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2025-03-02 11:29:47,706:INFO:main:>>>>>> stage Model Trainer stage started <<<<<<]
[2025-03-02 11:29:47,708:INFO:common:yaml file: config/config.yaml loaded successfully]
[2025-03-02 11:29:47,709:INFO:common:yaml file: params.yaml loaded successfully]
[2025-03-02 11:29:47,710:INFO:common:yaml file: schema.yaml loaded successfully]
[2025-03-02 11:29:47,710:INFO:common:created directory at: artifacts]
[2025-03-02 11:29:47,710:INFO:common:created directory at: artifacts/model_trainer]
[2025-03-02 11:29:47,718:INFO:main:>>>>>> stage Model Trainer stage completed <<<<<<

x==========x]
[2025-03-02 11:29:47,718:INFO:main:>>>>>> stage Model evaluation stage started <<<<<<]
[2025-03-02 11:29:47,720:INFO:common:yaml file: config/config.yaml loaded successfully]
[2025-03-02 11:29:47,720:INFO:common:yaml file: params.yaml loaded successfully]
[2025-03-02 11:29:47,721:INFO:common:yaml file: schema.yaml loaded successfully]
[2025-03-02 11:29:47,721:INFO:common:created directory at: artifacts]
[2025-03-02 11:29:47,721:INFO:common:created directory at: artifacts/model_evaluation]
[2025-03-02 11:29:48,183:INFO:common:json file saved at: artifacts/model_evaluation/metrics.json]
[2025-03-02 11:29:51,400:INFO:main:>>>>>> stage Model evaluation stage completed <<<<<<

x==========x]
